### PR TITLE
Add long description about this project

### PR DESCRIPTION
**Pull request by leplatrem**
I would love to receive feedback from @xorduna @Natim or @almet :) 

Thanks !!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,102 @@
...
+
+Using [Kinto](http://kinto.readthedocs.org/), this part requires no effort!
+
+That's why I took his idea and rebuilt a similar prototype in a few hours: to demo
+the value of *Kinto* in this context :) Hacking your project can now be **100% fun**!
+
+
+## How does it work ?
+
+1. A Telegram bot waits for messages, using [telepot](https://github.com/nickoala/telepot)
```
Comment (Natim) [inline README.md:41]: nit: Replace `1.` with `#.`

**Comment by Natim** [comment]
I have to say, I am impressed :+1:

File: README.md (hunk 1 above)
Comment (leplatrem) [inline README.md:41]: I previewed and it seems to work (i did a markdown btw)

**Comment by xorduna** [comment]
Hi Mathieu!

That's fun! I will try it next week, because this week I am very busy!

BTW, how can I get a kito user / key?

Thanks!

**Comment by Natim** [comment]
> BTW, how can I get a kito user / key?

You can use any BasicAuth login/password couple to create a unique UserId.

**Comment by leplatrem** [comment]
@xorduna cool ! good luck with London ;)

**Pull request merged**